### ISSUE TITLE
No error shown when gas limit set too low

### ISSUE THREAD

**Issue by lukaszfryc** [status-im/snt-voting#35]
### Description

*Type*: Bug

*Summary*: When gas limit is set to some low number, the transaction is not sent. In this case, the dapp should notify user what happened. Instead, the user is stuck on the sign to confirm screen with the sign button being disabled.

**Comment by Serhy** [comment]
@lukaszfryc this is an https://github.com/status-im/status-react/issues/5899 issue related to Wallet. We had a pop-up in the past notifying user (maybe with not really obvious explanation) about that. 

Not sure if Voting Dapp can detect the transactions sent with < 21000 gas limit, but if it's possible with any hook, (quick fix) @richard-ramos, lets do it. 

It's a **low priority** though.

**Comment by richard-ramos** [comment]
Ropsten:
https://ipfs.infura.io/ipfs/QmQBJBKNJAQhNrwkBMKsd4VbZpHTEgn5KdAD2kNfjUxE7N/#/

@lukaszfryc Sign button doesn't stay disabled now in this scenario.
@Serhy Seems like no transaction nor event is generated if the user sets a low gas limit, so I'm not able to capture this event.
@churik Latest version, with Metamask it shows the error direclty on Metamask, and no redirect happens.
![screenshot from 2018-11-08 14-44-45](https://user-images.githubusercontent.com/1106587/48220199-0fbc4980-e365-11e8-9167-14c8fe4dbe61.png)

**Comment by Graeme-Code** [comment]
Hey @lukaszfryc and @Serhy for your work on this issue. I don't expect people to go into advanced options to play around with GAS limits (unless they have a low amount of ETH and wanna stretch their usage). 

going back and forward is not great, but it does provide the user with a path to success if they encounter the issue. 

It's ok closing the issue, but would like to know why the behavior differs from metamask and status?

**Comment by Serhy** [comment]
If i'm correct, in Metamask validation error for inappropriate transaction values (Gas limit) is made in the transaction separate window, so Vote DApp page get notified only after transaction sent. While in Status browser, for this case, - we send transaction, appear on VoteDapp page, and only then realise that there is a gas limit which is < than 21000 (when "intrinsic gas too low" pop-up appears)

In MetaMask actually, if to close appeared transaction window (which is a separate tab) - 'Sign to confirm' also keep stay disabled.

Once https://github.com/status-im/status-react/issues/4569 is fixed the behavior in Status will be the same as in Metamask for this particular issue.

Closing this one!
